### PR TITLE
1087

### PR DESCRIPTION
**Pull request by tstanvir**
Add 1087 - Diablo tutorial

### CODE REVIEW

File: 1087/en.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# [http://lightoj.com/volume_showproblem.php?problem=1087](http://lightoj.com/volume_showproblem.php?problem=1087 "1087 - Diablo") #
```
Comment (Tahanima) [inline 1087/en.md:1]: Please add the problem title in this format - https://github.com/lightoj-dev/problem-tutorials/pull/11/files#diff-49d89fcefb3733849902cae167206bfef3d2d2fa1569a2abcc5b303e53f828b9R1

File: 1087/en.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+So,for prefix sum query and update we can use any range query data structure such as segment tree,Binary Indexed tree.
+As,prefix sum array is a monotonic array,we can binary search for the leftmost occurrence of the relative position the problem will ask for. 
+
+**Note: use fast I/O :3**
+
+----------
+
+
+**Code: C++:**
```
Comment (Tahanima) [inline 1087/en.md:65]: Please use the appropriate syntax for code blocks. Here is a reference - https://github.com/lightoj-dev/problem-tutorials/pull/22/files#diff-8b373101a9727d57ad4dc62e959037a667fd41e97ba2f9e1fa3cf56d2bdea2dbR22

File: 1087/en.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            }
+        }
+        else {
+            scanf("%d",&x);getchar();
+            vec[++en]=x;
+
+            update(en,1);
+        }
+    }
+
```
Comment (Tahanima) [inline 1087/en.md:125]: Please remove the unnecessary blank line.

File: 1087/en.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
+# [http://lightoj.com/volume_showproblem.php?problem=1087](http://lightoj.com/volume_showproblem.php?problem=1087 "1087 - Diablo") #
+
+
+Starting with an example to understand the problem better.
+
+Suppose, we have n people standing linearly and everyone has assigned with an id:
+6 5 3 2 1 --->id
+1 2 3 4 5 --->there relative position
+we will have two queries:
+
```
Comment (Tahanima) [inline 1087/en.md:10]: Please fix all your line breaks. Here's a reference - https://markdown-guide.readthedocs.io/en/latest/basics.html#line-return

**Review by Tahanima — CHANGES_REQUESTED** [review]
Hi, please resolve all the requested changes. Also, make sure to change your bn.md file accordingly as well.

**Comment by tstanvir** [comment]
hi , i have resolved the changes you have asked for . Can you recheck and let me know that it is okay , please ? Thanks.

**Comment by Tahanima** [comment]
Hi
1. You haven't followed the title style and format I've asked for.
2. Your writing has a mix of capital and small letters where it is expected to use capital letter. It looks very informal. Please fix that.
3. There's still unnecessary blank lines in your code.
4. Also, please follow the style we are using for headings such as **C++** and **Java**.
5. Your code is missing C++ header file, main function and some pre-requisites to get the code accepted. I would suggest you to include these missing parts.
6. You have mentioned about the use of BIT to solve the problem but you haven't explained the algorithm itself. Please provide link to a tutorial which explains this algorithm.

Please resolve the above mentioned issues.

**Comment by tstanvir** [comment]
Hello, sorry for the inconvenience. I've rechecked all the points you mentioned. Will you please let me know if there is further changes I need to resolve ? Thanks again.

**Comment by Tahanima** [comment]
Hi, 
Thank you for your contribution.

**Pull request merged**